component/src/main/java/io/siddhi/extension/io/file/FileHandler.java

```diff
@@ -0,0 +1,456 @@
+                                "define stream CreateFileStream (filepath string, length long, " +
+                                "last_modified string, status string);" +
+                                "@sink(type='log')" +
+                                "define stream FooStream (filepath string, length int, " +
+                                "last_modified string, status string); \n",
+
+                        description = "" +
+                                "Under above configuration, After monitoring has been started, If the specific " +
+                                "file given in the uri gets modified or deleted. an event is created" +
+                                " with the filepath, length, last modified date and status of the file. " +
+                                "Then that will be received by the FooStream..\n"
+                ),
+                @Example(
+                        syntax = "" +
+                                "@source(type='fileeventlistener',uri='file://abc/xyz',@map(type='json')) \n" +
+                                "define stream CreateFileStream (filepath string, length long, " +
+                                "last_modified string, status string);\n" +
+                                "@sink(type='log')" +
+                                "define stream FooStream (filepath string, length int, " +
+                                "last_modified string, status string); \n",
+
+                        description = "" +
+                                "Under above configuration, all the files in the specific uri " +
+                                "which has been  modified or created or deleted in the execution time " +
+                                "will be identified and displayed.\n After monitoring has been started, " +
+                                "If any files in the specific uri is newly created, modified or deleted.\n " +
+                                "an event is created with the filepath, length, last modified date " +
+                                "and status of the file. Then that will be received by the FooStream..\n"
+                ),
+                @Example(
+                        syntax = "" +
+                                "@source(type='fileeventlistener',uri='file://abc/xyz', monitoring.interval='200', " +
+                                "@map(type='json')) \n" +
+                                "define stream CreateFileStream (filepath string, length long, " +
+                                "last_modified string, status string);\n" +
+                                "@sink(type='log')" +
+                                "define stream FooStream (filepath string, length int, " +
+                                "last_modified string, status string); \n",
+
+                        description = "" +
+                                "Under above configuration, all the files in the specific uri \n" +
```

Comment:
Use the same description format

Reply:
fixed in https://github.com/siddhi-io/siddhi-io-file/pull/82/commits/3f9a2a0eb50655821dd322688aa034be33ef8860